src/qcodes/dataset/dond/do_2d.py

```diff
@@ -57,21 +34,7 @@ def do2d(
     num_points2: int,
     delay2: float,
     *param_meas: ParamMeasT,
-    set_before_sweep: bool | None = True,
-    enter_actions: ActionsT = (),
-    exit_actions: ActionsT = (),
-    before_inner_actions: ActionsT = (),
-    after_inner_actions: ActionsT = (),
-    write_period: float | None = None,
-    measurement_name: str = "",
-    exp: Experiment | None = None,
-    flush_columns: bool = False,
```

Comment:
flush_columns and set_before_sweep are not supported by dond yet. Need to figure out how to deal with that